build.gradle.kts

```diff
@@ -1,13 +1,13 @@
 plugins {
-    kotlin("jvm")
+    kotlin("jvm") version "1.3.40"
```

Comment:
I removed the version here because the template repo fails to compile since it also specifies a version. For some reason only one can specify it, and I was having trouble getting this one specified, and the other to leave it off